### PR TITLE
limit query getProphciesCompleted query result

### PR DESCRIPTION
**Pull request by juniuszhou**
we need to avoid the getPropheciesCompleted return too much prophecies in single query.

### CODE REVIEW

File: x/oracle/keeper/prophecy.go [hunk 1]
```diff
@@ -225,5 +229,5 @@ func (k Keeper) GetProphecyInfoWithScopeGlobalSequence(ctx sdk.Context,
 		globalSequence++
 		result = append(result, &prophecyInfo)
 	}
-
+	return result
```
Comment (jprupp) [inline x/oracle/keeper/prophecy.go:232]: Interesting that you didn't need this explicit return in the previous version, and you need it now.

**Review by jprupp — APPROVED** [review]
Looks good to me.

File: x/oracle/keeper/prophecy.go (hunk 1 above)
Comment (juniuszhou) [reply x/oracle/keeper/prophecy.go:232]: the loop changed from without any condition to a fixed times.

Comment (smartyalgo) [reply x/oracle/keeper/prophecy.go:232]: @jprupp Golang supports named return. that's why
https://stackoverflow.com/questions/45239409/empty-return-in-func-with-return-value-in-golang

**Pull request merged**